bio_pieces/parallel_blast.py

```diff
@@ -118,19 +135,40 @@ def parallel_diamond(inputfile, outfile, ninst, db, task, diamondoptions):
     dmnd_path = sh.which('diamond')
     if dmnd_path is None:
         raise ValueError("diamond is not in your path(Maybe not installed?)")
-    args = ['-u', '--pipe', '--block', '10', '--recstart', '>', '--cat']
-    args += sshlogins
-    args += [
+    diamond_cmd = [
         dmnd_path, task, '--threads', str(ninst), '--db', db, '--query', '{}',
-        '--compress', '0', '-a', outfile
-    ] + shlex.split(diamondoptions)
-    cmd = sh.Command('parallel')
-    run(cmd, args, inputfile, outfile)
+        '-a', '{}.{#}', ';', dmnd_path, 'view', '{}.{#}.daa'
```

Comment:
This is a little cryptic I would mention the need to do `diamond view` in the docstring, maybe mention what {#} is doing